### PR TITLE
Enhancement/devkit migrate

### PR DESCRIPTION
**Pull request by nolmoonen**
#### Deployment instructions
None.

#### What have I done
Migrated from Gradle to Devkit.

#### Testing
None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+* This should result in a directory structure `binary_distrib/<platform>/bin/lib/<platform>`. 
 
-## Run instructions
-* Run `intellji/runIde` Gradle script to debug the plugin in a sandbox.
+### Plugin
+* Open the project in [IntelliJ IDEA](<intellij link here>).
+* Add `.jar` files by selecting `File > Project Structure... > Project Settings/Libraries > New Project Library (+)` and selecting all `.jar` files from `binary_distrib/<platform>/bin`.
+* Build the plugin selecting `Build > Prepare Plugin Module "Exhibit" For Deployment`.
+* The built plugin is now located in the root under the name `Exhibit-mps.zip`.
+* Open the zip (do not unzip), create directory `browser_binaries` in root directory.
```
Comment (TijnBertens) [inline README.md]: I don't think it is the root directory, but rather the /lib/ directory

File: README.md [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
-The top level `<platform>` directory contains the distributions.
-* Create (top-level) directory ```binary_distrib``` in the project.
-* Copypaste the distribution directory (for example ```win64```) to the created directory. (`binary_distrib`) 
-Note that the directory specifying the platform, has another directory with the same name in its file tree. (for example it should look like: `binary_distrib\win64\bin\lib\win64`)
-* Open IntelliJ and select `File > New > Project from Existing Sources...` (or `Import Project` when on `Welcome to IntelliJ IDEA` screen), 
-and choose the project directory. 
-* Wait until Gradle finished configuring and downloading. (may take some time)
-* Run `intellji/buildPlugin` Gradle script to build the plugin. 
-The built plugin is now located in `build/distributions/exhibit-mps-1.0-SNAPSHOT.zip`
+The building process is rather complex. Note that these instructions are only for Windows.
```
Comment (TijnBertens) [inline README.md]: `rather complex` is not necessary, I would put something like 'It is important to follow the instructions closely' or something.

File: src/JCEFToolWindow.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        IdeaPluginDescriptor ideaPluginDescriptor = PluginManager.getPlugin(pluginId);
+
+        if (ideaPluginDescriptor == null) {
+            String errorMsg = "plugin descriptor not found! is plugin id \"" + pluginId + "\" correct?";
+            Logger.getInstance(JCEFToolWindow.class).error(errorMsg);
+            System.exit(1);
+        }
+
+        // CEF binary folder location
+        pluginPath = ideaPluginDescriptor.getPath() + "\\lib\\browser_binaries";
```
Comment (TijnBertens) [inline src/JCEFToolWindow.java]: Maybe put this constant in a variable

File: README.md (hunk 1 above)
Comment (nolmoonen) [reply README.md]: root of `.jar` file

File: README.md (hunk 2 above)
Comment (nolmoonen) [reply README.md]: changed

File: src/JCEFToolWindow.java (hunk 3 above)
Comment (nolmoonen) [reply src/JCEFToolWindow.java]: do not think that makes sense, shifted around declaration a bit

File: README.md [hunk 4]
```diff
@@ -1,26 +1,29 @@
...
+* Open the project in [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+* Add `.jar` files by selecting `File > Project Structure... > Project Settings/Libraries > New Project Library (+)` and selecting all `.jar` files from `binary_distrib/<platform>/bin`.
+* Build the plugin selecting `Build > Prepare Plugin Module "Exhibit" For Deployment`.
+* The built plugin is now located in the root under the name `Exhibit-mps.zip`.
+* Open the zip (do not unzip), create directory `browser_binaries` in `Exhibit-mps/lib`.
+* Copy **contents** of `binary_distrib/<platform>/bin/lib/<platform>` to `browser_binaries`.
 
-## Run instructions
-* Run `intellji/runIde` Gradle script to debug the plugin in a sandbox.
+The building process is now complete.
```
Comment (TijnBertens) [inline README.md]: Maybe specify here that the created .zip file is now a shippable plugin.

Comment (nolmoonen) [reply README.md]: okay

**Pull request merged**